collections/testtscripts.js

```diff
@@ -6,8 +6,9 @@ Testscripts.allow({
   }
 });
 
+
 Meteor.methods({
-  updateTicketStatus: function(ticket) {
+  updateTicketStatus: function(ticket, failReason) {
```

Comment:
I think passing in `failReason` can be removed since I don't see it being used in this method?